staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5147,6 +5147,8 @@ const (
 	NodePIDPressure NodeConditionType = "PIDPressure"
 	// NodeNetworkUnavailable means that network for the node is not correctly configured.
 	NodeNetworkUnavailable NodeConditionType = "NetworkUnavailable"
+	// NodeInitialized means that either node-controller or node-manager has initialized the node.
+	NodeInitialized NodeConditionType = "Initialized"
```

Comment:
it seems clearer to make this map *very* directly to the cloud provider uninitialized condition by naming it something like "CloudProviderUninitialized"

Reply:
NodeCloudProviderUninitialized could work too